### PR TITLE
144-[Mahesh|Rohit] Adds matchs any format of phoneNumber in the fuzzy…

### PR DESCRIPTION
**Pull request by rohit-yawalkar**
… logic

### CODE REVIEW

File: omod/src/main/java/org/bahmni/module/hip/web/service/ExistingPatientService.java [hunk 1]
```diff
@@ -25,7 +25,13 @@ public ExistingPatientService(PatientService patientService, ExistingPatientDao
     }
 
     public List<Patient> getMatchingPatients(String phoneNumber) {
-        return existingPatientDao.getPatientsWithPhoneNumber(phoneNumber);
+        String phoneNumberWithOutCode = phoneNumber.substring(phoneNumber.length() - 10);
+        String []formats = new String[4];
```
Comment (Nazeer-debug) [inline omod/src/main/java/org/bahmni/module/hip/web/service/ExistingPatientService.java:29]: 10,4 are Magical numbers, you can use constants

**Review by Nazeer-debug — CHANGES_REQUESTED** [review]

File: omod/src/main/java/org/bahmni/module/hip/web/service/ExistingPatientService.java (hunk 1 above)
Comment (swatigogia2020) [reply omod/src/main/java/org/bahmni/module/hip/web/service/ExistingPatientService.java:29]: agreed

**Review by swatigogia2020 — APPROVED** [review]

**Review by swatigogia2020 — CHANGES_REQUESTED** [review]
Make 4 and 10 as constants

**Review by Nazeer-debug — APPROVED** [review]

**Comment by rohityawalkar04** [comment]
@swatigogia2020, Removed magic numbers and added constants

**Review by tazeen7 — APPROVED** [review]

**Pull request merged**